src/components/popover/popover.tsx

```diff
@@ -201,7 +202,8 @@ export const Popover = forwardRef<PopoverRef, PopoverProps>((p, ref) => {
       if (!props.trigger) return
       setVisible(false)
     },
-    () => targetRef.current?.element
+    () => targetRef.current?.element,
```

Comment:
好像这里可以通过数组[传多个 target](https://ahooks.js.org/zh-CN/hooks/use-click-away#支持多个-dom-对象)，这样上面 105 行就不用手动处理 `stopPropagation` 了，我本地试了一下是 OK 的～

```suggestion
    [() => targetRef.current?.element, floatingRef],
```

Reply:
👌